testing/simapp/params/encoding.go

```diff
@@ -10,7 +10,7 @@ import (
 // This is provided for compatibility between protobuf and amino implementations.
 type EncodingConfig struct {
 	InterfaceRegistry types.InterfaceRegistry
-	Marshaler         codec.Codec
+	Codec             codec.Codec
```

Comment:
agree with this change 👍  but could make it separately to reduce the diffs

Reply:
happy to